bom/pom.xml

```diff
@@ -676,14 +672,12 @@
                 <groupId>io.debezium</groupId>
                 <artifactId>debezium-core</artifactId>
                 <version>${debezium.version}</version>
-                <scope>test</scope>
             </dependency>
             <dependency>
                 <groupId>io.debezium</groupId>
                 <artifactId>debezium-core</artifactId>
                 <version>${debezium.version}</version>
                 <type>test-jar</type>
-                <scope>test</scope>
```

Comment:
You remove the `test` scope here but you don't set it in other parts of the code. Do we really use these dependencies? Do we want to set the versions?

Reply:
They are used in the integration tests. (integration-tests/main/pom.xml)

Reply:
The code was already using the `test` scope.